src/components/accordion/Accordion.svelte

```diff
@@ -0,0 +1,27 @@
+<script>
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+  import { slide } from "svelte/transition";
```

Comment:
this accordion component is used in the new study card, though I've maintained the old, deprecated Accordion as well until we remove the old Ion experience.